src/main/java/net/snowflake/client/core/SFBaseSession.java

```diff
@@ -473,20 +495,35 @@ public HttpClientSettingsKey getHttpClientKey() throws SnowflakeSQLException {
                   "http",
                   userAgentSuffix,
                   gzipDisabled);
+          logger.info("Driver OCSP mode: {}, gzip disabled: {}, proxy protocol: {}," +
+                      " proxy host: {}, proxy port: {}, non proxy hosts: {}",
+                  ocspMode,
+                  gzipDisabled,
+                  proxyProtocol,
+                  httpProxyHost,
+                  proxyPort,
+                  combinedNonProxyHosts);
         } else {
           // Not enough parameters set to use the proxy.
-          logger.debug(
-              "http.useProxy={} but valid host and port were not provided. No proxy in use.",
-              httpUseProxy);
+          logger.warn(
+              "Failed parsing the proxy settings from JVM parameters as http.useProxy={}," +
+              " but valid host and port were not provided.",
+                  httpUseProxy);
+          logger.info("Drier OCSP mode: {}, gzip disabled: {} and no proxy",
+                  ocspMode,
+                  gzipDisabled);
           ocspAndProxyAndGzipKey =
-              new HttpClientSettingsKey(getOCSPMode(), userAgentSuffix, gzipDisabled);
+              new HttpClientSettingsKey(ocspMode, userAgentSuffix, gzipDisabled);
         }
       } else {
         // If no proxy is used or JVM http proxy is used, no need for setting parameters
         logger.debug("http.useProxy={}. JVM proxy not used.", httpUseProxy);
         unsetInvalidProxyHostAndPort();
+        logger.info("Snowflake driver OCSP mode: {}, gzip disabled: {} and no proxy",
```

Comment:
We start the previous logs with `Driver OCSP...` and here we have `Snowflake Driver...` - let's use the same naming

Reply:
Fixed.